roles/common/templates/nginx_mirror_vhost.conf.j2

```diff
@@ -10,28 +13,29 @@ server {
         root /usr/share/nginx/www;
     }
 
-    if ($bad_bot) { return 403; }
```

Comment:
Why remove the badbot filter? And if this is removed, I think the blacklist.conf should also be removed?